File: 1 - Beginner/1013 The Greatest/Python/main.py
def absolute(n):
    if (n<0):
        return n*-1
    return n

def MaiorAB(a,b):
    return (a+b+absolute(a-b))/2

a,b,c = input("").split()
a = int(a)
b= int(b)
c= int(c)
print(str(MaiorAB(MaiorAB(a,b),c))+" eh o maior")
    

File: 1 - Beginner/1013 The Greatest/C/Main.cpp
#include <stdio.h>
int abs(int number){
	if(number < 0){
		return number*-1;
	}
	return number;
}
int MaiorAB(int a, int b){
	return (a+b+abs(a-b))/2;
}
int main(void){
	int a,b,c;
scanf("%d %d %d",&a,&b,&c);
printf("%d eh o maior", MaiorAB(MaiorAB(a,b),c));
}


